docs/doxygen/pages.dox

```diff
@@ -0,0 +1,98 @@
+/**
+@mainpage Overview
+@anchor mqtt
+@brief Thread safe MQTT 3.1.1 client
```

Comment:
As this library is specific to FreeRTOS, would it be better to mention it clearly in the brief, so that customers can notice that coreMQTT is platform agnostic whereas the MQTT Agent is a thread-safe client for FreeRTOS
```suggestion
@brief Thread safe MQTT 3.1.1 client for FreeRTOS
```

Reply:
This library doesn't contain any FreeRTOS specific code any more, so I removed that from the description